c_sharp/image_classification/README.md

```diff
@@ -0,0 +1,51 @@
+# Image classification using an augmented ONNX model
+
+This samples uses the utilities in [ONNX Runtime extensions](https://github.com/microsoft/onnxruntime-extensions) to augment the MobileNetV2 model to include the data pre and post processing operations required by the model.
```

Comment:
>This samples uses the utilities in [ONNX Runtime extensions](https://github.com/microsoft/onnxruntime-extensions) to augment the MobileNetV2 model to include the data  [](http://example.com/codeflow?start=0&length=167)

nit: This sample...

I think this should also call out it's the PyTorch mobilenet (maybe with a link to it) as the pre-processing is specific to how that model was trained. 

TF does not use per-channel normalization so the pre-processing would be incorrect if it was TF mobilenet.